builder/oneandone/config.go

```diff
@@ -0,0 +1,87 @@
+package oneandone
+
+import (
+	"github.com/1and1/oneandone-cloudserver-sdk-go"
+	"github.com/mitchellh/mapstructure"
+	"github.com/mitchellh/packer/common"
+	"github.com/mitchellh/packer/helper/communicator"
+	"github.com/mitchellh/packer/helper/config"
+	"github.com/mitchellh/packer/packer"
+	"github.com/mitchellh/packer/template/interpolate"
+	"os"
+)
+
+type Config struct {
+	common.PackerConfig `mapstructure:",squash"`
+	Comm                communicator.Config `mapstructure:",squash"`
+
+	Token         string `mapstructure:"token"`
+	Url           string `mapstructure:"url"`
+	SSHKey        string
+	SSHKey_path   string              `mapstructure:"ssh_key_path"`
```

Comment:
I used it the same way here https://github.com/StackPointCloud/packer/blob/master/builder/profitbricks/config.go#L25 so I thought it was ok to use it again.